### PR TITLE
add ability to run terraform from run.py

### PR DESCRIPTION
**Pull request by azizova-leyla**
This now works:
cloud/shared/bin/run.py --command="terraform init"
cloud/shared/bin/run.py --command="terraform force-unlock LOCKID"

### CODE REVIEW

File: cloud/shared/bin/deploy.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from cloud.shared.bin.lib.config_loader import ConfigLoader
 
 
-def run(config: ConfigLoader):
+def run(config: ConfigLoader, params=None):
```
Comment (nbeloglazov) [inline cloud/shared/bin/deploy.py:8]: Given that run() now always called with 2 arguments and the second argument is always List[str], make it `run(config: ConfigLoader, params: List[str])`?

**Comment by nbeloglazov** [comment]
How will calling terraform look in the `civiform-deploy` repo?

**Comment by azizova-leyla** [comment]
I need to update civiform-deploy to make it work.
I will likely change` bin/run` to accept --command as argument that will be passed in here. Otherwise it gets tricky with multiword commands.

File: cloud/shared/bin/destroy.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 """
+from typing import List
 
 from cloud.shared.bin.lib import terraform
 from cloud.shared.bin.lib.config_loader import ConfigLoader
 from cloud.shared.bin.lib.setup_class_loader import get_config_specific_destroy
 
 
-def run(config: ConfigLoader):
+def run(config: ConfigLoader, params: List[str] = None):
```
Comment (nbeloglazov) [inline cloud/shared/bin/destroy.py:11]: nit: you don't need to provide = None given that second parameter is always passed.

**Review by nbeloglazov — APPROVED** [review]

**Pull request merged**